[#326] Front end log viewer

rappleb1 opened the pull request:
Closes #326 

Adds front-end data table for log viewing

rappleb1 committed:
working table no word wrap

rappleb1 committed:
cleaned up column widths

rappleb1 committed:
messaging and button enable

rappleb1 committed:
fix

ssung13 reviewed (approved):
Looks good to me! Probably be the best if we can shorten the long logs and make it expand when the user clicks on it.

rappleb1 committed:
test auto expand

rappleb1 commented:
> Looks good to me! Probably be the best if we can shorten the long logs and make it expand when the user clicks on it.

I think I have implemented this - please take a look and let me know how that looks!

ssung13 commented:
> > Looks good to me! Probably be the best if we can shorten the long logs and make it expand when the user clicks on it.
> 
> I think I have implemented this - please take a look and let me know how that looks!

Looks good!  Just one more thing: Would it be possible to make the expanded log collapse when clicked again? If this takes a lot of work, don't worry about it.

rappleb1 committed:
row colors and tap to close

rappleb1 committed:
formatting

rappleb1 commented:
> > > Looks good to me! Probably be the best if we can shorten the long logs and make it expand when the user clicks on it.
> > 
> > 
> > I think I have implemented this - please take a look and let me know how that looks!
> 
> Looks good! Just one more thing: Would it be possible to make the expanded log collapse when clicked again? If this takes a lot of work, don't worry about it.

Fixed! Also added alternating row colors for more visual clarity.
